microsoft-edge/visual-studio-code/debugger-for-edge.md

```diff
@@ -1,17 +1,25 @@
 ---
-description: How to debug Microsoft Edge from Visual Studio Code
-title: Debug Microsoft Edge from Visual Studio Code
+description: You can debug Microsoft Edge from Visual Studio Code using the Debugger Extension
+title: Debug Microsoft Edge in Visual Studio Code
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 07/26/2021
+ms.date: 11/08/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, debugger
 ---
-# Debug Microsoft Edge from Visual Studio Code
+# Debug Microsoft Edge in Visual Studio Code
 
-[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge which can launch the browser or attach to an already running browser.
+[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge, which can launch the browser or attach to an already running browser.
 
-This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from [Visual Studio Code](https://code.visualstudio.com).
+This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from Visual Studio Code.
 
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging) and [Browser debugging with Microsoft Edge DevTools integration in Visual Studio Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
+
+
+<!-- ====================================================================== -->
+## Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
+
+Send your feedback by [filing an issue](https://github.com/Microsoft/vscode-edge-devtools/issues/new) in the [vscode-edge-devtools repo](https://github.com/Microsoft/vscode-edge-devtools).
```

Comment:
This links to the devtools VS Code extension repo. The page about this extension is https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension
So that page should link to this repo (if it doesn't already do it).

The present page, however, is about the built-in JS debugger in VS Code. It is maintained by a different team, and its repo is this one: https://github.com/microsoft/vscode-js-debug

I think @codepo8 should decide what's the best thing to do here, but I would suggest:

```
## Get in touch with the js-debug team

Send your feedback by [filing an issue](https://github.com/microsoft/vscode-js-debug/issues/new) in the vscode-js-debug repository.
```

Reply:
Given that it's a short page, with no other h2 headings, I think there should be sentences with a link, but no h2 heading.

Reply:
changed